audio-download.py

```diff
@@ -86,27 +96,29 @@ def download(q, api, out_folder):
             return
         try:
             try:
-                out_path = download_name(r)
+                out_path = download_name(r, args.local_time)
             except ValueError as err:
                 print("error with {}: {}".format(r["id"], err))
                 continue
 
             print("downloading " + str(out_path))
-            iter_to_file(api.download_raw(r["id"]), str(out_folder / out_path))
+            iter_to_file(api.download_raw(r["id"]), str(args.out_folder / out_path))
         finally:
             q.task_done()
 
 
-def download_name(r):
+def download_name(r, local_time):
     dt = parsedate(r["recordingDateTime"])
+    if local_time:
+        dt = dt.astimezone(local_tz)
     device_name = r["Device"]["devicename"]
 
-    mime_type = r.get("fileMimeType")
+    mime_type = r.get("rawMimeType", "audio/mp4")
```

Comment:
Assume this default is intentional?
Perhaps should remove the if statement, unless we want to check for rawMimeType specifically being set to none?

Reply:
This shouldn't have been left in. Good catch.